### ISSUE TITLE
Install/upgrade tidalapi when mopidy-tidal pypi package is not pointing to latest version

### ISSUE THREAD

**Issue by tehkillerbee** [tehkillerbee/mopidy-tidal#51]

**Comment by tehkillerbee** [comment]
You can install the latest tidalapi package from PyPi manually:
```
pip3 install --upgrade --force-reinstall tidalapi
```

**Comment by fmarzocca** [comment]
I have just installed mopidy-tidal from pip:

`sudo pip3 install mopidy-tidal`

but it downgraded tidalApi:

```
Installing collected packages: tidalapi, mopidy-tidal
  Attempting uninstall: tidalapi
    Found existing installation: tidalapi 0.7.0rc1
    Uninstalling tidalapi-0.7.0rc1:
      Successfully uninstalled tidalapi-0.7.0rc1
Successfully installed mopidy-tidal-0.2.7 tidalapi-0.6.10
```

**Comment by BlackLight** [comment]
`tidalapi 0.7.0` has finally been released and the version on pypi updated. A PR is also ready to point to the latest version instead of the Github branch: https://github.com/tehkillerbee/mopidy-tidal/pull/77. Once the PR is merged, we can probably consider this issue fixed.

**Comment by tehkillerbee** [comment]
Fixed in #77
